components/802.15.4_RF/stm-s2lp-rf-driver/source/rf_configuration.c

```diff
@@ -181,5 +180,31 @@ uint32_t rf_conf_calculate_deviation(phy_modulation_index_e modulation_index, ui
     return deviation;
 }
 
+uint8_t rf_conf_calculate_pa_level_dbm(int8_t power_dbm)
+{
+#if MBED_CONF_S2LP_EXTERNAL_FRONTEND
+    if (power_dbm > 10) {
+        power_dbm = 10;
+    }
+#else
+    if (power_dbm > MAX_PA_VALUE) {
+        power_dbm = MAX_PA_VALUE;
+    }
+#endif
+
+    if (power_dbm < MIN_PA_VALUE) {
+        power_dbm = MIN_PA_VALUE;
+    }
+
+    uint8_t pa_level_value = 0;
+    if (power_dbm > 14) {
```

Comment:
should the 14 and 29 use defined values ?

Reply:
I'll create define values